view/frontend/templates/js/replacejs.phtml

```diff
@@ -1714,16 +1717,48 @@ if (!$block->isSaveCartInSections()) { ?>
                 if (waitingForResolvingPromises && (localCallNumber === countConfigureCalls)) {
                     customerData.reload(['boltcart'], true);
                 }
-            }, 5000);
+            }, 10000);
```

Comment:
On my [very slow] test site + with local bolt sometimes it takes 6 seconds. Anyway
- in the worst case [when user click add to cart and we check how many seconds need to update bolt cart] we have pretty much the same latency but with better UI (promises)
- in general case [user goes from one page to another] we win because we don't update bolt cart anymore, get it from local storage.
- we have this check just in case, it never should work